iroh-js/src/lib.rs

```diff
@@ -70,7 +70,7 @@ pub fn start_metrics_collection() -> Result<()> {
     iroh_metrics::core::Core::try_init(|reg, metrics| {
         metrics.insert(iroh::metrics::MagicsockMetrics::new(reg));
         metrics.insert(iroh::metrics::NetReportMetrics::new(reg));
-        metrics.insert(iroh::metrics::PortmapMetrics::new(reg));
+        metrics.insert(iroh::metrics::PortmapMetrics::default());
```

Comment:
what changed here?

Reply:
Not totally sure? It said there was no `new` method for `PortmapMetrics`?